modules/eks/actions-runner-controller/main.tf

```
@@ -0,0 +1,192 @@
+        "ecr:GetLifecyclePolicyPreview",
+        "ecr:GetRepositoryPolicy",
+        "ecr:InitiateLayerUpload",
+        "ecr:ListImages",
+        "ecr:PutImage",
+        "ecr:PutImageScanningConfiguration",
+        "ecr:PutImageTagMutability",
+        "ecr:PutLifecyclePolicy",
+        "ecr:StartImageScan",
+        "ecr:StartLifecyclePolicyPreview",
+        "ecr:TagResource",
+        "ecr:UntagResource",
+        "ecr:UploadLayerPart",
+      ]
+      resources = ["*"]
+    }
+  ]
+
+  s3_iam_policy_statements = length(var.s3_bucket_arns) > 0 ? [
+    {
+      sid = "AllowS3Actions"
+      actions = [
+        "s3:PutObject",
+        "s3:GetObject",
+        "s3:DeleteObjectVersion",
+        "s3:ListBucket",
+        "s3:DeleteObject",
+        "s3:PutObjectAcl"
+      ]
+      resources = flatten([
+        for arn in var.s3_bucket_arns :
+        [arn, "${arn}/*"]
+      ])
+    },
+  ] : []
+
+  iam_policy_statements = concat(local.default_iam_policy_statements, local.s3_iam_policy_statements)
+}
+
+data "aws_partition" "current" {
+  count = local.enabled ? 1 : 0
+}
+
+data "aws_ssm_parameter" "github_token" {
+  count = local.enabled ? 1 : 0
+
+  name            = var.ssm_github_token_path
+  with_decryption = true
+}
+
+module "actions_runner_controller" {
+  source  = "cloudposse/helm-release/aws"
+  version = "0.6.0"
+
+  name                 = "" # avoids hitting length restrictions on IAM Role names
+  chart                = var.chart
+  repository           = var.chart_repository
+  description          = var.chart_description
+  chart_version        = var.chart_version
+  kubernetes_namespace = var.kubernetes_namespace
+  create_namespace     = var.create_namespace
+  wait                 = var.wait
+  atomic               = var.atomic
+  cleanup_on_fail      = var.cleanup_on_fail
+  timeout              = var.timeout
+
+  eks_cluster_oidc_issuer_url = module.eks.outputs.eks_cluster_identity_oidc_issuer
+
+  service_account_name      = module.this.name
+  service_account_namespace = var.kubernetes_namespace
+
+  iam_role_enabled = true
+
+  iam_policy_statements = local.iam_policy_statements
+
+  values = compact([
+    # hardcoded values
+    file("${path.module}/resources/values.yaml"),
+    # standard k8s object settings
+    yamlencode({
+      fullnameOverride = module.this.name,
+      serviceAccount = {
+        name = module.this.name
+      },
+      resources = var.resources
+      rbac = {
+        create = var.rbac_enabled
+      }
+      githubWebhookServer = {
+        enabled = var.webhook.enabled
+        ingress = {
+          enabled = var.webhook.enabled
+          hosts = [
+            {
+              host = local.webhook_host
+              paths = [
+                {
+                  path     = "/"
+                  pathType = "Prefix"
+                }
+              ]
+            }
+          ]
+        }
+      },
+      authSecret = {
+        enabled = true
```

If we are not including SOPS support now, then set 
```suggestion
        enabled = true
        create  = true
```